axelrod/match.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from random import random
```

Comment:
Yeah good shout. I'll change that.

On Thu, 18 Feb 2016, 20:38 Marc Harper <notifications@github.com> wrote:

> In axelrod/match.py
> <https://github.com/Axelrod-Python/Axelrod/pull/461#discussion_r53378761>:
>
> > @@ -1,4 +1,6 @@
> >  # -*- coding: utf-8 -*-
> > +from random import random
>
> I usually prefer to import the module in the case that you need a function
> of the same name to avoid later confusion:
>
> import random
> r = random.random()
> # This still works elsewhere in the file
> ri = random.randint()
>
> from randomimport random
> r = random()
> # This doesn't work anymore
> ri = random.randint()
>
> Same for the datetime module. That's just my preference, I find it keeps
> me from making little mistakes later when I try to modify a larger file.
> Also note that some projects like numpy avoid this and have names like
> numpy.random.rand instead of numpy.random.random.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/461/files#r53378761>.
>